### PR TITLE
scheme: add page

### PR DESCRIPTION
**Pull request by alex0112**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/scheme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+> MIT Scheme language interperter and REPL (interactive shell)
+
+- Open an interactive shell (REPL):
+
+`scheme`
+
+- Run a scheme program (with no REPL output):
+
+`scheme --quiet < script.scm`
```
Comment (agnivade) [inline pages/common/scheme.md]: script.scm in tokens (`{{}}`)

File: pages/common/scheme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Load a scheme program into the REPL:
+
+`scheme --load {{script.scm}}`
+
+- Load scheme expressions into the REPL:
+
+`scheme --eval "(define foo 'x)"`
```
Comment (agnivade) [inline pages/common/scheme.md]: expression in tokens too

File: pages/common/scheme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Suppress the startup report of versions and copyrights, and the valediction.
+  
+  `scheme --quiet`
```
Comment (agnivade) [inline pages/common/scheme.md]: Additional space before the example

File: pages/common/scheme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Load a scheme program into the REPL:
+
+`scheme --load {{script.scm}}`
+
+- Load scheme expressions into the REPL:
+
+`scheme --eval "(define foo 'x)"`
+
+- Suppress the startup report of versions and copyrights, and the valediction.
```
Comment (agnivade) [inline pages/common/scheme.md]: Probably we do not need to specify what quiet mode does. How about just -  "Open the REPL in quiet mode" ?

File: pages/common/scheme.md (hunk 2 above)
Comment (alex0112) [reply pages/common/scheme.md]: Should that be `{{"(define foo 'x)"}}` or `"{{(define foo 'x)}}"` ?

**Comment by alex0112** [comment]
@agnivade Made those changes.

**Comment by alex0112** [comment]
The Travis Build is saying that command descriptions should end in a period, but I consistently see most descriptions end with a colon.  Which is correct?

**Comment by pxgamer** [comment]
Example descriptions should end in a colon. Youve done that correctly. However, line 3 should be:
```
> MIT Scheme language interperter and REPL (interactive shell).
```
(Ending in a period)

**Comment by alex0112** [comment]
@pxgamer I think everything has been addressed and I've got a passing build.  Let me know if there's anything else.

**Comment by agnivade** [comment]
Thank you @alex0112

**Pull request merged**